hyperdata/src/normalize.ts

```diff
@@ -71,14 +71,17 @@ export function contentFrom(meta: any, eager?: boolean): Content {
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function generativeFrom(meta: any): GenArt | undefined {
-  const uri = meta.generativeUri || meta.generatorUri
+  const uri = meta.generativeUri || meta.generatorUri || meta.generative_uri
 
   if (!uri) {
     return undefined
   }
 
   const hash = meta.previewHash || meta.iterationHash
-  const previewParam = meta.previewParam || 'fxhash'
+  const previewParam =
+    meta.previewParam ||
+    (meta.external_url?.includes('koda') && 'koda') ||
+    'fxhash'
```

Comment:
Preview param of this 

```
ipfs://bafybeihxrnjmqs7po3jw3p37csyscvgxuwg2khow5zm46liutxq466q45m/?hash=0x044852b2a670ade5407e78fb2863c51de9fcb96542a07186fe3aeda6bb8a116
```

is `hash`

```suggestion
  const previewParam =
    meta.previewParam || 'hash'
```

Reply:
updated